README.md

```diff
@@ -87,11 +89,11 @@ Now, build and deploy Derek:
 
 ```
 $ docker build -t derek .
-$ faas-cli -action build -name derek -image derek -fprocess=./derek
+$ faas-cli deploy --name derek --image derek --fprocess=./derek
 ```
 
 **Testing**
 
-Create a label of "no-dco" within every project you want Derek to help you with.
+Try creating a label of "no-dco" within a project you want Derek to help you with.
```

Comment:
Not sure this line needed to change

Reply:
I changed that because reading it stepwise it appears as an instruction; that is, in order to enable Derek to manage/monitor a repo the user should manually create a `no-dco` label within that repo.  I think its actually a lead-in to the following statement - the change was meant to move more towards a description of the following action.  

No big deal to me, it can come out.